CIPs/0019.md

```diff
@@ -0,0 +1,49 @@
+# CIP [0019]: JWT signer for DID-JWT
+
+- Date: 2020-09-09
+- Author: @daoauth
```

Comment:
hi @daoauth  thank you for submitting this CIP, I would like you to update it to follow the newly activated CIP process which has it's newly updated template here: https://github.com/celo-org/celo-proposals/blob/master/CIPs/cip-template.md

Also, if the CIP is in Draft status, I'd be able to merge the proposal into the repo.

Reply:
yes. I updated. but However, I couldn't write a few items. And I don't know how to write the Category and Track.

Reply:
@prestwich would this be Ring 2?

Reply:
@daoauth for Track, it's "Standards"